fluentd/files/fluentd.service

```diff
@@ -10,6 +10,8 @@ ExecReload=/usr/local/bin/fluentd.sh reload
 Restart=always
 PIDFile=/var/run/fluentd.pid
 TimeoutStartSec=1
+User=fluentd
+Group=fluentd
```

Comment:
This should be templated to pull in the user and group that gets rendered through the map data, which also means that it would need to be moved from `files` to `templates` and have file.managed references updated accordingly.